spacy/tests/pipeline/test_spancat.py

```diff
@@ -0,0 +1,38 @@
+from spacy.language import Language
+from spacy.training import Example
+from spacy.util import fix_random_seed
+
+
+TRAIN_DATA = [
+    ("Who is Shaka Khan?", {"entities": [(7, 17, "PERSON")]}),
+    ("I like London and Berlin.", {"entities": [(7, 13, "LOC"), (18, 24, "LOC")]}),
+]
+
+
+def make_get_examples(nlp, spans_key="spans"):
+    train_examples = []
+    for t in TRAIN_DATA:
+        eg = Example.from_dict(nlp.make_doc(t[0]), t[1])
+        eg.reference.spans[spans_key] = list(eg.reference.ents)
+        train_examples.append(eg)
```

Comment:
We'll need a proper way to read these custom spans into `doc.spans` without passing through `doc.ents`.

And for documentation examples I wouldn't use the key `"spans"` ;-)